hub/include/P2PController.h

```diff
@@ -6,13 +6,36 @@
 #include "SystemCallWrapper.h"
 #include "HubController.h"
 
+static QString p2p_messages[] = {
+  QObject::tr("The connection with environment is not established."),
+  QObject::tr("Can't to connect container."),
+  QObject::tr("Container doesn't have desktop."),
+  QObject::tr("Press this button to ez-ssh to container."),
+  QObject::tr("Press this button to ez-desktop to container.")
+};
+
+enum p2p_message_code_t {
+  CANT_JOIN_SWARM = 0,
+  CANT_CONNECT_CONTAINER,
+  NO_DESKTOP,
+  CLICK_EZ_SSH,
+  CLICK_EZ_DESKTOP
+};
```

Comment:
I don't like the fact that you use no array indexes and this may lead to problems in the future with product support.